context/app/static/js/components/Detail/ProvTabs.jsx

```diff
@@ -36,13 +37,38 @@ function TabPanel(props) {
 
 
 function ProvTabs(props) {
-  const { provData, assayMetadata } = props;
+  const { uuid, assayMetadata, entityEndpoint } = props;
   const classes = useStyles();
-  const [open, setOpen] = React.useState(0);
 
+  const [open, setOpen] = React.useState(0);
   const handleChange = (event, newValue) => {
     setOpen(newValue);
   };
+
+  const [provData, setProvData] = React.useState(null);
+  React.useEffect(() => {
+    fetch(
+      `${entityEndpoint}/entities/${uuid}/provenance`,
+      {
+        headers: {
+          Authorization: `Bearer ${readCookie('nexus_token')}`,
+        },
+      },
+    )
+      .then((response) => {
+        if (!response.ok) {
+          console.error('Prov API failed', response);
+          return null;
+        }
+        return response.json();
+      })
+      .then(
+        (responseProvData) => {
+          setProvData(responseProvData);
+        },
+      );
```

Comment:
I lean async/await over promise chains where possible. Do we care about being consistent with that?

Reply:
We can figure this out later, but some sort of progress component while loading?

Reply:
async implemented. Filed #365 for please wait. (It's typically below the fold, and loads fast, so I don't think this is pressing.)